### PR TITLE
Feature public feed

### PR DESCRIPTION
**Pull request by mcardosog**

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/MessageFeedServlet.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    response.setContentType("application/json");
+
+    List<Message> messages = datastore.getAllMessages();
+    Gson gson = new Gson();
+    String json = gson.toJson(messages);
+
+    response.getOutputStream().println(json);
+  }
+}
```
Comment (ctrokid) [inline src/main/java/com/google/codeu/servlets/MessageFeedServlet.java]: Nitpicky: add a new line at the end of the file? 😄

**Comment by ctrokid** [comment]
This looks good to me (LGTM) outside of my one nitpicky comment.

One tip for next time if I'm not the reviewer:
- Add a description for the PR. Your commit messages are nice, but the PR description will make it easier to review.

**Comment by ctrokid** [comment]
Oh, actually one more thing. Do you see how the base branch says `master`? Can you change that to `develop` before you merge? I can explain more offline if you're curious.

**Comment by mcardosog** [comment]
I already change the branch to develop. I believe that it needs to be approved again.

**Review by ctrokid — APPROVED** [review]
I would have you fix the new line at the end of the file, but that's super nit picky and was more to show you the PR review process.

LGTM.

File: src/main/java/com/google/codeu/servlets/MessageFeedServlet.java (hunk 1 above)
Comment (mcardosog) [reply src/main/java/com/google/codeu/servlets/MessageFeedServlet.java]: Ups! I will do it know.

**Review by mcardosog — COMMENTED** [review]
New line added at the end of the file.

**Review by mcardosog — COMMENTED** [review]
Styling updated. Space needed before "+"

**Pull request merged**